<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>device</class>
 <widget class="QWidget" name="device">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_18" native="true">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 8px;
background-color:rgb(248, 249, 251);
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="0,1,0">
        <item>
         <widget class="QCheckBox" name="check_device">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRepeatDelay">
           <number>0</number>
          </property>
          <property name="autoRepeatInterval">
           <number>0</number>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="text_device">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="change_btn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color:rgb(240, 241, 242);
color:rgb(255, 255, 255);
font-weight: 600;
border-radius: 8px;
padding: 5px 15px;
outline: 0px;
}
QPushButton:hover{
background-color: rgb(125, 143, 202);
}
</string>
          </property>
          <property name="icon">
           <iconset resource="resources/icons.qrc">
            <normaloff>:/icon/editing.png</normaloff>:/icon/editing.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>36</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
